index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="ko">
+	<head>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link rel="stylesheet" href="style.css">
+		<title>판다마켓</title>
+	</head>
+	<body>
+		<!-- header section / 로고, 로그인 -->
+		<header>
+			<div class="banner">
+				<div class="banner-main">
+					<a class="header-logo" href="index.html">
+						<img src="./images/header-logo.png" alt="로고">
+					</a>
+					<a id="login" href="login.html">로그인</a>
+				</div>
+			</div>
+		</header>
+		<!-- main section -->
+		<main>
+			<!-- top section -->
+			<section>
+				<div class="top-section">
+					<div class="top-content">
+						<div class="top-text">
+							<div class="top-title">
+								<h2>일상의 모든 물건을<br>
+									거래해 보세요
+							</h2>

```

Comment:
💊 제안

현재 코드에서는 여는 태그`<h2>`와 닫는 태그`</h2>`의 indent가 서로 달라 가독성이 조금 떨어질 수 있습니다.
태그 간 계층 구조를 명확하게 표현하기 위해 아래처럼 작성하시는 것을 추천드려요~

```suggestion
								<h2>일상의 모든 물건을<br>거래해 보세요</h2>
```